ceph_iscsi_config/target.py

```diff
@@ -385,10 +385,15 @@ def load_config(self):
             # there could/should be multiple tpg's for the target
             for tpg in self.target.tpgs:
                 self.tpg_list.append(tpg)
-                ip_address = list(tpg.network_portals)[0].ip_address
-                gateway_name = self._get_gateway_name(ip_address)
-                if gateway_name:
-                    self.tpg_tag_by_gateway_name[gateway_name] = tpg.tag
+                network_portals = list(tpg.network_portals)
+                if len(network_portals) > 0:
```

Comment:
NIT: we could get away with just:

  if network_portals:

Or maybe even:

  if tpg.network_portals:

Which if empty or None would result in a False. But otherwise looks good :)

Reply:
Thanks! Changed